test/acceptance/ramp-buckets-regressions.test.js

```diff
@@ -53,3 +53,122 @@ describe('conditional ramp-buckets', function () {
     });
   });
 });
+
+describe('Buckets calculation', function () {
+  function createCartoCSS (numBuckets) {
+    return [
+      '#layer {',
+      '  marker-width: ramp([population], range(1, 100), quantiles(' + numBuckets + '));',
+      '  marker-fill: #EE4D5A;\n  marker-fill-opacity: 0.9;\n',
+      '  marker-allow-overlap: true; \n  marker-line-width: 1; \n',
+      '  marker-line-color: #FFFFFF; \n  marker-line-opacity: 1; \n',
+      '}'
+    ].join('\n');
+  }
+
+  var scenarios = [
+    {
+      desc: 'Buckets: 1',
+      numBuckets: 1,
+      dataSourceStats: { min_val: 1, max_val: 100, avg_val: 50 },
+      ramp: [1],
+      expectedBuckets: [
+        {
+          'filter': {
+            'type': 'range',
+            'start': 1,
+            'end': 100
+          },
+          'value': 1
+        }
+      ],
+      expectedStats: {
+        filter_avg: 50
+      }
+    },
+    {
+      desc: 'Buckets: 2',
+      numBuckets: 2,
+      dataSourceStats: { min_val: 1, max_val: 100, avg_val: 50 },
+      ramp: [1, 2],
+      expectedBuckets: [
+        {
+          'filter': {
+            'type': 'range',
+            'start': 1,
+            'end': 1
+          },
+          'value': 1
+        },
+        {
+          'filter': {
+            'type': 'range',
+            'start': 1,
+            'end': 100
+          },
+          'value': 100
+        }
+      ],
+      expectedStats: {
+        filter_avg: 50
+      }
+    },
+    {
+      desc: 'Buckets: 3',
+      numBuckets: 3,
+      dataSourceStats: { min_val: 1, max_val: 100, avg_val: 50 },
+      ramp: [1, 2, 3],
+      expectedBuckets: [
+        {
+          'filter': {
+            'type': 'range',
+            'start': 1,
+            'end': 1
+          },
+          'value': 1
+        },
+        {
+          'filter': {
+            'type': 'range',
+            'start': 1,
+            'end': 2
+          },
+          'value': 50.5
+        },
+        {
+          'filter': {
+            'type': 'range',
+            'start': 2,
+            'end': 100
+          },
+          'value': 100
+        }
+      ],
+      expectedStats: {
+        filter_avg: 50
+      }
+    }
+  ];
+
+  scenarios.forEach(function (scenario) {
+    it(scenario.desc, function (done) {
+      var datasource = new DummyDatasource(function () {
```

Comment:
A `DummyDatasource` should receive a function (`getter`) with the following signature:
```
function (column, buckets, method) {
  return [1, 2, 3, ...., 100];
}
```
And should return an array with data, numbers in this case.

Reply:
Ok, I got it. It also works with: 
```
{
  ramp: scenario.ramp,
  strategy: '>',
  stats: scenario.dataSourceStats
}
```

Sorry for the noise